### ISSUE TITLE
Installer Won't Let Me Install

### ISSUE THREAD

**Issue by ashyashton** [riperiperi/FreeSO#214]
I looked for help on the Discord and received no response. I did the complete installation multiple times, and every time, it tells me FreeSO is ready to play. I hit play, and it says I need FreeSO and The Sims Online installed to play. I click the button to install The Sims Online and it says that it's detected that The Sims Online is already installed, so I set that as my game installation folder again. I click the button to install FreeSO, and it says I need The Sims Online and OpenAL installed, but both of them were obviously installed in the complete installation.

Basically, I can't play the game at all because the installer, for some reason, doesn't wanna register that I've installed it. I have tried restarting the installer/launcher and it still doesn't work. Can someone please help me figure this out before I just give up?

**Comment by riperiperi** [comment]
The launcher is not covered by this repository, it is hosted at:
https://github.com/ItsSim/fsolauncher

The launcher relies on accessing the registry to detect installation for both FreeSO and TSO. Maybe an antivirus is preventing it from doing that?
